src/corelibs/U2View/src/ov_sequence/GSequenceLineView.cpp

```diff
@@ -682,7 +682,7 @@ qint64 GSequenceLineViewRenderArea::coordToPos(const QPoint& coord) const {
     int x = qBound(0, coord.x(), width());
     const U2Region& visibleRange = view->getVisibleRange();
     double scale = getCurrentScale();
-    qint64 pos = qRound64((double)visibleRange.startPos + x / scale);
+    qint64 pos = (double)visibleRange.startPos + x / scale;
```

Comment:
Use qFloor to avoid the implicit type conversion

Reply:
OK